README.md

```diff
@@ -61,20 +61,32 @@ To learn more about the different rule types, their guarantees and how they are
 #### Carrying out software supply chain steps
 
 ##### in-toto-run
-`in-toto-run` executes the passed command and records the path and hash of the passed *materials* - files before command execution - and *products* - files after command execution and optionally stores them together with the command's *byproducts* (e.g: return value, stdout or stderr) to a link file (`<step-name>.link`), signed with the functionary's key.
+`in-toto-run` executes the passed command and records the path and hash of
```

Comment:
This paragraph might need a revision.  It's one long sentence and I find it difficult to determine its main purpose.

Can you state it's main purpose in a shorter sentence.  For example,  "in-toto-run generates link metadata for the given command."

Reply:
See issue #159